src/components/CourseChecklist/CourseChecklist.scss

```diff
@@ -31,14 +37,14 @@ $success-color: theme-color("success");
   }
 }
 
-.font-large {
-  font-size: $font-size-lg;
-}
-
 .font-small {
   font-size: $font-size-sm;
 }
 
+.font-large {
```

Comment:
I wonder if these font sizes should be put in `SFE.scss`. Any chance they'd be useful to reuse going forward?

Reply:
They seem generic enough that moving them up to the top level makes sense to me. You can either move it now, or wait until it's necessary.